### PR TITLE
Update dockerfile with pa11y-ci

### PR DESCRIPTION
**Pull request by tjheffner**
also removed some lines that are now included by default in the base alekzonder/puppeteer:1 image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,22 +11,14 @@ LABEL maintainer="Phase2 <outrigger@phase2technology.com>" \
...
 
 # Let's get pa11y v5 in here.
-RUN yarn global add pa11y@beta
-
-# Add user so we don't need --no-sandbox.
-RUN groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
-    && mkdir -p /home/pptruser/Downloads \
-    && chown -R pptruser:pptruser /home/pptruser \
-    && chown -R pptruser:pptruser /screenshots \
-    && chown -R pptruser:pptruser /usr/local/share/.config/yarn/global/node_modules
```
Comment (grayside) [inline Dockerfile:26]: By dropping these permissions, can the pptuser still take screenshots?

File: Dockerfile [hunk 2]
```diff
@@ -11,22 +11,14 @@ LABEL maintainer="Phase2 <outrigger@phase2technology.com>" \
...
+RUN yarn global add pa11y@5
+RUN yarn global add pa11y-ci@2
```
Comment (grayside) [inline Dockerfile]: Can we have one of the following to minimize the number of layers?
```
RUN yarn global add pa11y@5 pa11y-ci@2
```
or
```
RUN yarn global add pa11y@5 && yarn global add pa11y-ci@2
```

Also, is it the case we need to explicitly add pa11y? We can't invoke it if it's a dependency of pa11y-ci?

File: Dockerfile [hunk 3]
```diff
@@ -11,22 +11,14 @@ LABEL maintainer="Phase2 <outrigger@phase2technology.com>" \
...
 
 USER pptruser
 
-ENTRYPOINT ["dumb-init", "--", "pa11y"]
+ENTRYPOINT ["dumb-init", "--"]
```
Comment (grayside) [inline Dockerfile]: I like the flexibility here, but if we remove pa11y as part of the entrypoint, the -h flag in command will be an error.

I think we'll need to remove the CMD statement entirely.

This approach leaves us with:

```
docker run --rm outrigger/pa11y pa11y <url>
```

or

```
docker run --rm outrigger/pa11y pa11y-ci
```
a bit redundant, but we can work with that.

**Review by grayside — CHANGES_REQUESTED** [review]
Overall looking good. I have a couple minor tweaks I'd like to see taken care of.

File: Dockerfile (hunk 3 above)
Comment (grayside) [reply Dockerfile]: From sidebar chat, maybe CMD should be "bash -c"?

File: Dockerfile (hunk 2 above)
Comment (tjheffner) [reply Dockerfile]: I have confirmed that pa11y needs to be explicitly added. If pa11y-ci gets added globally, that doesn't give access to pa11y globally

File: Dockerfile (hunk 1 above)
Comment (tjheffner) [reply Dockerfile:26]: I believe so, but I have not tested the screenshot capability myself. These permissions are now included in the starting dockerfile here: https://hub.docker.com/r/alekzonder/puppeteer/~/dockerfile/

**Review by grayside — APPROVED** [review]

**Pull request merged**